OrbitLinuxTracing/PerfEvent.h

```diff
@@ -369,6 +369,18 @@ class TaskRenamePerfEvent : public TracepointPerfEvent {
   }
 };
 
+class TracepointEventPidTidTimeCpu : public TracepointPerfEvent {
```

Comment:
This should follow the name convention of the other classes in this hierarchy, i.e., `...PerfEvent`. Maybe `GenericTracepointPerfEvent`?

Reply:
Yes, you are right. I will change the name. Thanks!